packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -1137,18 +1138,18 @@ describe('Meter', () => {
 
   describe('#batchObserver', () => {
     it('should create a batch observer', () => {
-      const measure = meter.createBatchObserver('name', () => {});
-      assert.ok(measure instanceof Metric);
+      const measure = meter.createBatchObserver(() => {});
```

Comment:
Nit: because it is not a measure anymore I would rename all `measure` into `batcher`

```suggestion
      const batcher = meter.createBatchObserver(() => {});
```